langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -90,6 +97,13 @@ public isolated function abs(int n) returns int = @java:Method {
 
 # Returns sum of zero or more int values.
 #
+# ```ballerina
+# int:sum(10, 20, 30, 40) ⇒ 100
+# 
+# int[] marks = [50, 65, 78, 95];
+# int total = int:sum(...marks) ⇒ 288
+# ```
```

Comment:
We discussed to add another example like

```
# decimal d = 21.2;
# d.sum(10.5, 21, 32.4) ⇒ 85.1
```
in the float and decimal, shall we add the same in here. wdyt?